mooncake-integration/CMakeLists.txt

```diff
@@ -31,12 +35,20 @@ set(PYTHON_PACKAGE_NAME "mooncake")
 
 pybind11_add_module(engine ${SOURCES} ${CACHE_ALLOCATOR_SOURCES}
     transfer_engine/transfer_engine_py.cpp 
+    ../mooncake-transfer-engine/src/transport/coro_rpc_connector/cororpc_interface.cpp
+    ../mooncake-transfer-engine/src/transport/coro_rpc_connector/cororpc_communicator.cpp
```

Comment:
Why need to add these files here?

Reply:
if we want to use coro_rpc_connector as a submodule of mooncake transfer engine, we have to compile it here